manifests/g/Guhan/GoodCommit/1.0.1/Guhan.GoodCommit.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Guhan.GoodCommit
+PackageVersion: 1.0.1
+InstallerType: portable
+Commands:
+- good-commit
+Installers:
+- InstallerUrl: https://github.com/Gukkey/good-commit-cli/releases/download/v.1.0.1/good-commit-cli.exe
+  Architecture: neutral

```

Comment:
Instead of using `Architecture: neutral`, the preferred way would be to have separate nodes for all architectures supported by the package. If this package truly supports x86, x64, arm & arm64, you should create separate nodes as:

```
# Created using wingetcreate 1.6.1.0
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json

PackageIdentifier: Guhan.GoodCommit
PackageVersion: 1.0.1
InstallerType: portable
Commands:
- good-commit
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Oracle.JDK.22
    MinimumVersion: 22.0.2.0
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/Gukkey/good-commit-cli/releases/download/v.1.0.1/good-commit-cli.exe
  InstallerSha256: 289c03b55a63fdce70553a4a1c6d19373e833b008c8c00f4b267fb0591a624c4
- Architecture: x86
  InstallerUrl: https://github.com/Gukkey/good-commit-cli/releases/download/v.1.0.1/good-commit-cli.exe
  InstallerSha256: 289c03b55a63fdce70553a4a1c6d19373e833b008c8c00f4b267fb0591a624c4
- Architecture: arm
  InstallerUrl: https://github.com/Gukkey/good-commit-cli/releases/download/v.1.0.1/good-commit-cli.exe
  InstallerSha256: 289c03b55a63fdce70553a4a1c6d19373e833b008c8c00f4b267fb0591a624c4
- Architecture: arm64
  InstallerUrl: https://github.com/Gukkey/good-commit-cli/releases/download/v.1.0.1/good-commit-cli.exe
  InstallerSha256: 289c03b55a63fdce70553a4a1c6d19373e833b008c8c00f4b267fb0591a624c4
ManifestType: installer
ManifestVersion: 1.6.0

```

Reply:
I have not tested or can verify it will work in other architecture other than x64, so does mentioning x64 means only people having that architecture can install the CLI?

Reply:
> so does mentioning x64 means only people having that architecture can install the CLI?

Yes

Reply:
Okay, for now I will adding only x64. I will test on other architecture with help of others and add them on later version. Thank you.